static_report/src/components/shared/Charts/HistogramChart.tsx

```diff
@@ -89,7 +96,7 @@ export function HistogramChart({
     grid: { display: false },
     ticks: {
       callback(val, index) {
-        return newLabels[index];
+        return hideAxis ? null : newLabels[index];
```

Comment:
![image](https://user-images.githubusercontent.com/11186480/186554895-ae1f667c-2ce5-478b-ab87-d51a49bc52d0.png)
Would passing it to this config option be better?
Or does it screw up the scaling?

Reply:
I didn't notice this, I think it is better than above.